tests/main/classic-confinement-not-supported/task.yaml

```diff
@@ -28,11 +30,13 @@ execute: |
     if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
         EXPECTED_TEXT="classic confinement requires snaps under /snap or symlink from /snap to /var/lib/snapd/snap"
     fi
+    #shellcheck disable=SC2015
```

Comment:
For reference: https://github.com/koalaman/shellcheck/wiki/SC2015

Reply:
Is it telling us that the || true is not needed in practice?
Unless `$(...)` inherits `set -e` that is, but does it?

Reply:
I'll drop the whole `.. && exit 1 || true` as it makes little sense here.